app/models/agents/email_digest_agent.rb

```diff
@@ -59,13 +57,13 @@ def check
               content_type: interpolated['content_type'],
               groups: groups
             ).deliver_now
-            log "Sent digest mail to #{recipient} with events [#{ids}]"
+
+            log "Sent digest mail to #{recipient}"
           rescue => e
-            error("Error sending digest mail to #{recipient} with events [#{ids}]: #{e.message}")
+            error("Error sending digest mail to #{recipient}: #{e.message}")
             raise
           end
         end
-        self.memory['queue'] = []
         self.memory['events'] = []
```

Comment:
Let's leave `self.memory.delete('queue') if self.memory.key?('queue')` for legacy cleanup.